src/config/cache-config.interface.ts

```diff
@@ -13,6 +13,8 @@ export interface CacheConfig extends Config {
   serverSide: {
     // Debug server-side caching.  Set to true to see cache hits/misses/refreshes in console logs.
     debug: boolean,
+    // List of headers to restore from the cache hit
+    headers: string[],
```

Comment:
Similar to above, I think this makes more sense if it's the "List of headers to *cache*".  That way we are not using up unnecessary cache space.

Reply:
done